pkg/odo/cli/init/init.go

```diff
@@ -113,6 +116,47 @@ func (o *InitOptions) Validate() error {
 // Run contains the logic for the odo command
 func (o *InitOptions) Run(ctx context.Context) (err error) {
 
+	devfileObj, _, err := o.run(ctx)
+	if err != nil {
+		return err
+	}
+
+	exitMessage := fmt.Sprintf(`
+Your new component '%s' is ready in the current directory.
+To start editing your component, use 'odo dev' and open this folder in your favorite IDE.
+Changes will be directly reflected on the cluster.`, devfileObj.Data.GetMetadata().Name)
+
+	// show information about `odo deploy` command only if the devfile has kind: deploy command defined.
+	commands, _ := devfileObj.Data.GetCommands(common.DevfileOptions{
+		CommandOptions: common.CommandOptions{
+			CommandGroupKind: v1alpha2.DeployCommandGroupKind,
+		},
+	})
+	if len(commands) != 0 {
+		exitMessage += "\nTo deploy your component to a cluster use \"odo deploy\"."
+	}
```

Comment:
Can you replace this with `libdevfile.HasDeployCommand()` method?
```suggestion
	if libdevfile.HasDeployCommand(devfileObj.Data) {
		exitMessage += "\nTo deploy your component to a cluster use \"odo deploy\"."
	}
```